src/data/treasury-data.ts

```diff
@@ -14,30 +14,30 @@ export const TOKENS: {
     name: string
   }
 } = {
-  everipedia: {
+  IQ: {
     id: 'everipedia',
     icon: BraindaoLogo,
     name: 'IQ',
   },
-  weth: {
+  WETH: {
     id: 'weth',
     icon: WETH,
     name: 'WETH',
   },
-  frax: {
+  FRAX: {
     id: 'frax',
     icon: Fraxswap,
     name: 'FRAX',
   },
-  'frax-share': {
+  FXS: {
     id: 'frax-share',
     icon: FXS,
     name: 'FXS',
   },
```

Comment:
still missing sfrxETH

Reply:
@Simplypheyie96 , I need an Icon for sfrxETH.

Reply:
something like this should be fine (in svg if needed) https://app.beefy.com/static/media/sfrxETH.77143d0f.png

Reply:
> something like this should be fine (in svg if needed) https://app.beefy.com/static/media/sfrxETH.77143d0f.png
I already gave him something similar to the rest of the lp token icons we have on the page. do you think we should change it? 
<img width="370" alt="image" src="https://user-images.githubusercontent.com/99387231/204296828-139a205e-9666-4b54-9ed1-3b5a9c736ea4.png">

Reply:
sfrxETH is not an LP token ( https://www.pcmag.com/encyclopedia/term/lp-token#:~:text=(Liquidity%20Pool%20or%20Liquidity%20Provider,redeemed%20for%20the%20original%20tokens. ) , that would be wrong. sfrxETH its ether staked on frax platform to get yield. But the underlying token is ETH not ETH + FRAX

Reply:
okay.
i understand